eng/Subsets.props

```diff
@@ -581,6 +581,11 @@
     <ProjectToBuild Include="$(MonoProjectRoot)\wasm\workloads.proj" Category="mono" />
   </ItemGroup>
 
+  <!-- Libraries native set -->
+  <ItemGroup Condition="$(_subset.Contains('+libs.native+'))">
+    <ProjectToBuild Include="$(SharedNativeRoot)libs\build-native.proj" Category="libs" />
+  </ItemGroup>
+
```

Comment:
Instead of moving libs.native up, can you move the host subsets to build right before libs.pretest? If we are going to need to run tooling on the outputs of libs.sfx to build the host, the host subsets should come after the libs.sfx subset.